### DISCUSSION TITLE
How to provide billing details to PaymentElement in Flutter Web

### DISCUSSION THREAD

**Discussion by tapizquent** [flutter-stripe/flutter_stripe#1518] [Q&A]
**Describe the bug**
Using the CardField, there is a way to specify the billing details, but when using the PaymentElement, there's not param to allow for this. How do we capture the billing details when using this element?

**To Reproduce**
1. Create a PaymentElement in Flutter Web
2. Try to set the Billing details

**Expected behavior**
Should be able to specify the billing details for the PaymentElement.

**Smartphone / tablet**
 - Device: Chrome
 - OS: MacOS
 - Package version: latest
 - Flutter version: latest

**Comment by ankitmiyan30** [comment]
any update on this? Facing a similar challenge and due to that payment is failing.

**Comment by tapizquent** [comment]
@jonasbark Any info on this? I see this was moved to a discussion, but seems like a bug tbh
